### PR TITLE
Add link to methods in yaml selector docs

### PR DESCRIPTION
**Pull request by bennieregenold7**
## Description & motivation

The docs on yaml selectors mention using `method: value` syntax, but it doesn't link out to the available methods. This adds that link so it's easier to find when looking through the docs.

### CODE REVIEW

File: website/docs/reference/node-selection/yaml-selectors.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ This simple syntax does not support any operators or `exclude`.
 
 ### Full YAML
 
-This is the most thorough syntax, which can include graph and set operators.
+This is the most thorough syntax, which can include graph and set operators. Checkout the list of available [methods](methods).
```
Comment (mirnawong1) [inline website/docs/reference/node-selection/yaml-selectors.md]:
```suggestion
This is the most thorough syntax, which can include graph and set operators. 

Review [methods](/reference/node-selection/methods) for the entire list of available.  
```

**Review by mirnawong1 — DISMISSED** [review]
hey @bennieregenold7 thank you so much! one small suggestion and approved for you.

**Comment by bennieregenold7** [comment]
@mirnawong1 - thank you for the suggestion! I made the commit and confirmed with the deploy preview that it works as intended. Can you please re-approve when you get a chance?

File: website/docs/reference/node-selection/yaml-selectors.md [hunk 2]
```diff
@@ -60,7 +60,9 @@ This simple syntax does not support any operators or `exclude`.
 
 ### Full YAML
 
-This is the most thorough syntax, which can include graph and set operators.
+This is the most thorough syntax, which can include graph and set operators. 
+
+Review [methods](/reference/node-selection/methods) for the entire list of available.
```
Comment (mirnawong1) [inline website/docs/reference/node-selection/yaml-selectors.md]:
```suggestion
Review [methods](/reference/node-selection/methods) for the available list.
```

**Review by mirnawong1 — APPROVED** [review]
ready for you @bennieregenold7 !! ✨

**Pull request merged**